packages/react/src/components/CardEditor/CardEditForm/CardEditFormItems/TableCardFormItems/TableCardFormContent.test.jsx

```diff
@@ -49,7 +50,7 @@ describe('TableCardFormContent', () => {
     expect(screen.queryByText('temperature')).toBeDefined();
     fireEvent.click(screen.queryByText('temperature'));
     // the selection state of the box should be updated
-    expect(screen.queryAllByTitle('Clear all selected items')).toHaveLength(1);
+    expect(document.querySelector('[aria-label="Clear Selection"]')).toBeTruthy();
```

Comment:
This is due to `translateWithId` being a mocked function

Reply:
Can this be scoped to the container?